src/commcare_cloud/environment/paths.py

```diff
@@ -4,11 +4,24 @@
 import yaml
 from memoized import memoized_property, memoized
 
-REPO_BASE = os.path.expanduser('~/.commcare-cloud/repo')
-ANSIBLE_DIR = os.path.join(REPO_BASE, 'ansible')
-ENVIRONMENTS_DIR = os.environ.get('COMMCARE_CLOUD_ENVIRONMENTS', os.path.join(REPO_BASE, 'environments'))
-FAB_DIR = os.path.join(REPO_BASE, 'fab')
-FABFILE = os.path.join(REPO_BASE, 'fabfile.py')
+
+def get_virtualenv_path():
+    return os.path.dirname(sys.executable)
```

Comment:
I think this should be, although not sure it's actually needed, given the comment below on `get_virtualenv_site_packages_path`.
```py
return os.environ["VIRTUAL_ENV"]
```

Reply:
Replaced `get_virtualenv_site_packages_path` with the built-in (thanks!), but I still do need to keep this function around, because it's used in a couple other places.

I didn't know about `VIRTUAL_ENV`, but that's actually exactly what I was trying to work _around_ with this way of fetching that dir. That env variable is set when the user is has entered the virtual environment, but _not_ set when the user runs an executable within the virtualenv `bin` directory directly.

Also, I'm realizing from your comment that this function is poorly named. It returns the path to the `bin` directory, so I'm going to name it accordingly.

Reply:
A bit of context helps so much! Maybe add a docstring to this function that explains why it's not using `os.environ["VIRTUAL_ENV"]` (because it's not expected to be set).